node/src/reactor/validator.rs

```diff
@@ -581,10 +581,11 @@ impl reactor::Reactor for Reactor {
                                     return Effects::new();
                                 }
                             };
-                            Event::Storage(storage::Event::GetDeployForPeer {
-                                deploy_hash,
-                                peer: sender,
-                            })
+                            todo!() // WTF?
```

Comment:
This should be reimplemented without the direct access to the event type, either through a request or via effect builder.